.github/workflows/doc.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync OAS to ReadMe
+on:
+  push:
+    branches:
+      - main
+      - readme-sync
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: readmeio/github-readme-sync@v2
```

Comment:
@atymchuk could you please clone this action? others should be there already.

Reply:
@BupycHuk please change all actions to the percona-platform.

Reply:
cloned!